src/components/TextLink.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {ForwardedRef, forwardRef, KeyboardEventHandler, MouseEventHandler, ReactElement} from 'react';
+import {GestureResponderEvent, Text as RNText, TextStyle} from 'react-native';
+import styles from '@styles/styles';
+import * as Link from '@userActions/Link';
+import CONST from '@src/CONST';
+import Text from './Text';
+
+type TextLinkProps = {
+    /** Link to open in new tab */
+    href: string;
+
+    /** Text content child */
+    children: ReactElement;
+
+    /** Additional style props */
+    style?: TextStyle;
+
+    /** Overwrites the default link behavior with a custom callback */
+    onPress?: () => void;
```

Comment:
This is more accurate?
```suggestion
    onPress?: (event?: GestureResponderEvent) => void;
```

Reply:
No, `openLink` doesn't accept arguments:
```
        if (onPress) {
            onPress();
            return;
        }

        Link.openExternalLink(href);
    };
```